src/main/java/lotto/domain/Ranking.java

```diff
@@ -28,10 +28,13 @@ public static Ranking findRanking(int count, boolean hasBonusNumber) {
     }
 
     private static boolean checkCountAndBonusNumber(Ranking ranking, int count, boolean hasBonusNumber) {
-        if (count == 5) {
+        if (ranking.count != count) {
+            return false;
+        }
+        if (count == SECOND.getCount()) {
```

Comment:
```suggestion
        if (count == SECOND.count) {
```
해당 클래스 내의 로직이니 getter를 거치지 않아도 될 것 같아요 :)